### PR TITLE
refactor: update scripts / scritps names along with dockerfiles start cmd 

### PR DESCRIPTION
**Pull request by TeamWolfyta**

### CODE REVIEW

**Review by cubismod — APPROVED** [review]
looks good (cubis from Discord)

**Comment by VictoriqueMoe** [comment]
adding context to this. It was useful to rename the start scripts to something more logical, as my English is not great.

**Review by VictoriqueMoe — APPROVED** [review]

File: 1-starter/package.json [hunk 1]
```diff
@@ -8,9 +8,9 @@
   "scripts": {
     "build": "tsc",
     "build:changelog": "npx @discordx/changelog --root=src",
-    "dev": "node --loader ts-node/esm/transpile-only src/main.ts",
-    "serve": "node build/main.js",
-    "start": "nodemon --exec node --loader ts-node/esm/transpile-only src/main.ts"
+    "dev": "ts-node-esm src/main.ts",
```
Comment (oceanroleplay) [inline 1-starter/package.json:11]: why?

Comment (TeamWolfyta) [reply 1-starter/package.json:11]: previous one was unnecessarily long when there was already a script provided by ts-node aka ts-node-esm

Comment (VictoriqueMoe) [reply 1-starter/package.json:11]: looks like it was just renamed.

Comment (oceanroleplay) [reply 1-starter/package.json:11]: that's good, I didn't knew

**Pull request merged**